site/src/pages/release-notes/1.26.0.mdx

```diff
@@ -0,0 +1,199 @@
+---
+date: 2023-10-20
+---
+
+## 🌟 New features
+
+- You can now add an `EventLoopGroup` to a server, a virtual host and a service to run a service with it.
+  #4099 #5233
+  ```java
+  final EventLoopGroup serviceEventLoopGroup =
+    EventLoopGroups.newEventLoopGroup(8, "service-worker");
+  Server.builder()
+        .eventLoopGroup(serviceEventLoopGroup)
+        .service(myService) // myService is run with serviceEventLoopGroup.
```

Comment:
Does this work? I thought it's something like this:

```java
Server
  .builder()
  .route()
  ...
  .eventLoopGroup(...)
  .service(myService)
  ...
```

Reply:
That's correct. 😓 Fixed. 👍